charts/sonarqube/templates/_pod.tpl

```diff
@@ -60,8 +60,20 @@ spec:
     - name: ca-certs
       image: {{ default (include "sonarqube.image" $) .Values.caCerts.image }}
       imagePullPolicy: {{ .Values.image.pullPolicy  }}
-      command: ["sh"]
-      args: ["-c", "cp -f \"${JAVA_HOME}/lib/security/cacerts\" /tmp/certs/cacerts; if [ \"$(ls /tmp/secrets/ca-certs)\" ]; then for f in /tmp/secrets/ca-certs/*; do keytool -importcert -file \"${f}\" -alias \"$(basename \"${f}\")\" -keystore /tmp/certs/cacerts -storepass changeit -trustcacerts -noprompt; done; fi;"]
+      command:
+        - sh
+        - -c
+        - |
+          #!/bin/sh
+          cp -f "${JAVA_HOME}/lib/security/cacerts" /tmp/certs/cacerts
+          if [ "$(ls /tmp/secrets/ca-certs)" ]; then
+            for f in /tmp/secrets/ca-certs/*; do
+              keytool -importcert -file "${f}" -alias "$(basename "${f}")" -keystore /tmp/certs/cacerts -storepass changeit -trustcacerts -noprompt
+            done
+          fi
+          keytool -importkeystore -srckeystore /tmp/certs/cacerts -destkeystore /tmp/certs/cacerts.p12 -deststoretype PKCS12 -srcstorepass changeit -deststorepass changeit
+          openssl pkcs12 -info -in /tmp/certs/cacerts.p12 -out /tmp/certs/cacerts.pem -nodes -passin pass:changeit
```

Comment:
just to confirm, here the goal is to duplicate the created cert bundle but export it as pem format so that it can be used by curl?

Side question, Curl accept no other format ?
Side-side question: for java, we need a full structure, but I think for curl, the certificate will be an extra one on top of the system one, meaning we probably don't need the whole copy import logic. wdyt ? That might simplify the code

Reply:
Question 1/2: Not sure about all formats it supports. For sure, it does not support the keystore made for Java apps (JKS)

Question 3: What would be your suggestion here? Just add the file and load in the chart when needed?

Reply:
After discussing it, we decided not to convert the keystore but just add the certificate we need when downloading the plugin.